app/models/support_user.rb

```diff
@@ -10,7 +12,7 @@ def self.load_from_session(session)
     dfe_sign_in_user = DfESignInUser.load_from_session(session)
     return unless dfe_sign_in_user
 
-    SupportUser.find_by(dfe_sign_in_uid: dfe_sign_in_user.dfe_sign_in_uid)
+    SupportUser.find_by(dfe_sign_in_uid: dfe_sign_in_user.dfe_sign_in_uid, discarded_at: nil)
```

Comment:
Should we use the `kept` scope here instead of `discarded_at`?

Reply:
@duncanjbrown good point - amended here https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1517/files#diff-e24635ab750d5f5d41f7b82858b9ec65R15

Reply:
> @duncanjbrown good point - amended here https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1517/files#diff-e24635ab750d5f5d41f7b82858b9ec65R15

looks good 👌